src/libraries/System.Text.Json/src/ILLink/ILLink.Substitutions.xml

```diff
@@ -0,0 +1,8 @@
+<linker>
+  <assembly fullname="System.Text.Json">
+    <type fullname="System.Text.Json.JsonSerializer">
+      <method signature="System.Boolean get_IsReflectionEnabledByDefault()" body="stub" value="false" 
+              feature="System.Text.Json.JsonSerializer.IsReflectionEnabledByDefault" featurevalue="false"/>
```

Comment:
As part of updating that document, we also need to define an MSBuild property name (see the existing switches for examples). This name will be user facing as well, since that is how the switch can be controlled by the user - for example the `<InvariantGlobalization>` MSBuild property.

Possible MSBuild property names:

* IsJsonSerializerReflectionEnabledByDefault 
* JsonSerializerEnableReflectionByDefault
* JsonSerializerIsReflectionEnabledByDefault

Reply:
I'm tending towards `JsonSerializerIsReflectionEnabledByDefault` since it reflects the feature switch name.

Reply:
I think that works. 👍